src/compress.py

```diff
@@ -0,0 +1,48 @@
+class CompressString:
+	def __init__(self, string='', letter=None, compression='', count=0):
+		self.string = string
+		self.compression = compression
+		self.letter = letter
+		self.count = count
+	
+	def _is_letter(self):
+		return self.letter.isalpha()
+	
+	def _is_previous_letter_same(self):
+		if self.compression == '':
+			return False
+		prev_letter = self.compression[-1]
+		return prev_letter == self.letter
+	
+	def _add_letter(self):
+		self.compression += self.letter
+	
+	def _add_count(self):
+		self.compression += str(self.count)
+	
+	def _increment_count(self):
+		self.count += 1
```

Comment:
Consider whether you want a method or would it be better to use the expression directly. Sometimes you still want a named function that acts like a label. 

There's no right or wrong; it's about readability and code clarity.